util/main.scss

```diff
@@ -30,10 +27,6 @@
 		}
 	}
 
-	.n-util-visually-hidden {
-		@include nUtilsVisuallyHidden;
-	}
-
 	.no-js .n-util-hide-no-js,
 	.js .n-util-hide-js,
 	.core .n-util-hide-core,
```

Comment:
would getting rid of the n-util namespace entirely be possible, so these go to n-ui-hide-no-js etc

Reply:
oh yeah, good shout. I'll update and add it to the migration guide